common/scripts/update-imodeljs-native-version.sh

```diff
@@ -13,14 +14,15 @@ checkfail() {
   fi
 }
 
-while getopts "v:" options; do
+while getopts "v:b:" options; do
   case "${options}" in
     v) AddonVersion=${OPTARG} ;;
+    b) AddonBranch=${OPTARG} ;;
     ?) usage ; exit 1 ;;
   esac
 done
 
-if [ "$AddonVersion" = "" ]; then
+if [ "$AddonVersion" = "" ] || [ "$AddonBranch" = "" ]; then
```

Comment:
The script I believe gets called by `./upgrade-imodeljs-native.sh`, you might want to double check this scripts functionality to ensure this won't break it. upgrade-imodejs-native.sh shouldn't really be used anymore because now we have automated jobs for this, but it might be worth making this optional and default it to master if left empty.

Reply:
yeah defaulting to master seems reasonable, just need to make sure we actually pass a branch in the release branches

Reply:
Will check it right away (default to master if left empty)